lib/src/components/NxSegmentedButton/NxSegmentedButton.scss

```diff
@@ -88,11 +86,11 @@
         border-radius: 4px;
```

Comment:
I bet this could/should be `calc(var(--nx-border-radius) - var(--nx-focus-outline-inset))`

Reply:
> I bet this could/should be `calc(var(--nx-border-radius) - var(--nx-focus-outline-inset))`

Fixed here [7062fd7](https://github.com/sonatype/sonatype-react-shared-components/pull/1038/commits/7062fd7b606ff6ee1864fdb40d44e8132ff9c94a)